Neos.Neos/Classes/Domain/Service/NodeSearchService.php

```diff
@@ -64,17 +64,26 @@ public function findByProperties($term, array $searchNodeTypes, Context $context
         $searchResult = array();
         $nodeTypeFilter = implode(',', $searchNodeTypes);
 
-        if (preg_match(NodeIdentifierValidator::PATTERN_MATCH_NODE_IDENTIFIER, $term) !== 0) {
-            $nodeByIdentifier = $context->getNodeByIdentifier($term);
-            if ($nodeByIdentifier !== null && $this->nodeSatisfiesSearchNodeTypes($nodeByIdentifier, $searchNodeTypes)) {
-                $searchResult[$nodeByIdentifier->getPath()] = $nodeByIdentifier;
+        if (is_string($term)) {
+            $searchTerm = [$term];
+        } else {
+            $searchTerm = $term;
+        }
+
+        foreach ($searchTerm as $term) {
+            if (preg_match(NodeIdentifierValidator::PATTERN_MATCH_NODE_IDENTIFIER, $term) !== 0) {
+                $nodeByIdentifier = $context->getNodeByIdentifier($term);
+                if ($nodeByIdentifier !== null && $this->nodeSatisfiesSearchNodeTypes($nodeByIdentifier, $searchNodeTypes)) {
+                    $searchResult[$nodeByIdentifier->getIdentifier()] = $nodeByIdentifier;
```

Comment:
Returning nodes by identifier (instead of the path) is quite a "breaking" behavior. Can you elaborate why you changed that?

Reply:
For a stupid reason that I can't explain ;)

And yes, this is a bugfix and need to move to a lower branch, will push a new version soonish